README.md

```diff
@@ -65,8 +65,14 @@ You will need:
 
 To build from the command line:
 1. Visual Studio VS2019 (Community Edition) with C#, C++, .Net Core
-2. .NET SDK 2.2.402 (https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-2.1.4-windows-x64-installer)
-- runtimes ???
+2. .NET SDKs (https://dotnet.microsoft.com/download/visual-studio-sdks)
+
+```
+2.1.816 (* - LTS Version)
+3.1.410 (* - LTS version)
+5.0.301 (* - this is specified in the global.json)

```

Comment:
the 5.x is not LTS or latest, But the 6.x Previews are.

Reply:
6.x is preview (5.x is latest) - once it comes out of preview I'll consider supporting it; I have been previously burnt by trying to support "preview" .net frameworks I have since learnt to wait.